### PR TITLE
Give health score a shot

### PR DESCRIPTION
**Pull request by filmaj**
Testing out this alpha GitHub Action we are working on

### CODE REVIEW

File: .github/workflows/deno-ci.yml [hunk 1]
```diff
@@ -38,3 +38,11 @@ jobs:
         with:
           file: ./lcov.info
           token: ${{ secrets.CODECOV_TOKEN }}
+                        
+      - name: Report health score
+        uses: slackapi/slack-health-score@v0.0.2
+        with:
+          codecov_token: ${{ secrets.FILS_CODECOV_API_TOKEN }}
```
Comment (WilliamBergamin) [inline .github/workflows/deno-ci.yml:45]: Is this the correct token to use?

**Review by WilliamBergamin — CHANGES_REQUESTED** [review]
This is sick 💯 but not sure about the token naming, is this for testing?

File: .github/workflows/deno-ci.yml (hunk 1 above)
Comment (filmaj) [reply .github/workflows/deno-ci.yml:45]: Yeah I just wanted to make it clear which human user the API token belongs to (in case I e.g. get hit by a bus)

**Comment by filmaj** [comment]
You can click the check to see a breakdown of how the score is calculated.

**Pull request merged**